logstash-core/lib/logstash/config/defaults.rb

```diff
@@ -6,6 +6,14 @@ module LogStash module Config module Defaults
 
   extend self
 
+  def input
```

Comment:
Well nothing against that, as we have no clear guidelines on that I can not really argument :smile_cat:, It does the work, so no problem on it.